tensorzero-core/src/variant/chat_completion.rs

```diff
@@ -50,6 +50,83 @@ pub struct ChatTemplates {
     pub assistant: Option<TemplateWithSchema>,
 }
 
+impl ChatTemplates {
```

Comment:
can we move this into its own or some utilities file? this one is huge and this type gets used in many variants